src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -316,29 +307,34 @@ public Command ArmDefaultCommand(Supplier<Double> armVelocity, Supplier<Double>
 	
 	public Command ToggleBrakeModes() {
 		return this.runOnce(() -> {
-			if (leaderArmMotor.getIdleMode() == IdleMode.kBrake) {
```

Comment:
Won't these over write the entire existing configuration, not just the break mode.

Reply:
This won't overwrite existing configuration, since the configuration is set with `ResetMode.kNoResetSafeParameters`.